calculator.rb

```diff
@@ -0,0 +1,116 @@
+#this method will take in numbers and operator
+#from the user, and return the operation as in
+#a calculator
+
+
+
+#this checks to make sure the user input is an operator
+def really_equation(operator)
+  special = "-+/%*^"
+  regex = /[#{special.gsub(/./){|char| "\\#{char}"}}]/
```

Comment:
Interesting use of a regular expression.  You could have also done `regex = /[\+\-\/\%\^]/`  skipping the `gsub`.